ext/native/gfx/gl_common.h

```diff
@@ -4,6 +4,9 @@
 #include <OpenGLES/ES3/gl.h>
 #include <OpenGLES/ES3/glext.h>
 #elif defined(USING_GLES2)
+#ifdef ODROID
```

Comment:
Maybe we can just always define `GL_GLEXT_PROTOTYPES` (ignoring any ODROID define)?  It won't be great for any static PPSSPP linux builds, but should be fine elsewhere I think?

-[Unknown]

Reply:
I did that, and build failed on Android then, the thing is, each vendor has it's own implementation of this and has it's own headers

Reply:
changed but I think travis will fail on Android again

Reply:
I see.  We could try fighting that battle with `#ifndef GL_NV_framebuffer_blit` in gl3stub, but maybe it is better to use the ODROID define as you suggested.  @hrydgard what do you think?

-[Unknown]